src/Symfony/Component/Console/Helper/ProgressBar.php

```diff
@@ -43,6 +43,7 @@ class ProgressBar
     private $formatLineCount;
     private $messages = array();
     private $overwrite = true;
+    private $firstRun = true;
```

Comment:
Sorry about that, but I preferred the previous version where it was a simple var in the overwrite method.